witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -241,10 +241,10 @@ checkFairCV =
         }
 
 checkFairCVPDF :: forall ctx m. WitnessWorkMode ctx m => ByteString -> m FairCVAndCheckResult
-checkFairCVPDF pdf = do
+checkFairCVPDF (LBS.fromStrict -> pdf) = do
     let maybeFairCV = do
             fairCVencoded <- Pdf.project (Pdf.MaxSearchLength (Just 2048)) (Pdf.PDFBody pdf)
-            Aeson.decode $ BS.fromStrict fairCVencoded
+            Aeson.decode $ LBS.fromStrict fairCVencoded
```

Comment:
And here we can use `Aeson.decodeStrict`

Reply:
Right, nice catch